### PR TITLE
docs(README): Update readme to remove beta status

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,8 @@
 # Deis Monitor v2
-
-[![Build Status](https://travis-ci.org/deis/monitor.svg?branch=master)](https://travis-ci.org/deis/monitor)
```
Comment (bacongobbler) [inline README.md:3]: for anyone else reviewing: travis is no longer used in this repo.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,8 @@
...
-[![Docker Repository on Quay](https://quay.io/repository/deisci/monitor/status "Docker Repository on Quay")](https://quay.io/repository/deisci/monitor)
```
Comment (bacongobbler) [inline README.md:4]: is there a link we should update this to?

Comment (jchauncey) [inline README.md:4]: no because we dont produce a monitor image. instead we do 3 images (grafana, telegraf, and influx)

File: telegraf/README.md [hunk 3]
```diff
@@ -1,9 +1,6 @@
 # Telegraf
 [![Docker Repository on Quay](https://quay.io/repository/deisci/telegraf/status "Docker Repository on Quay")](https://quay.io/repository/deisci/telegraf)
 
-## Description
```
Comment (bacongobbler) [inline telegraf/README.md:4]: Should we just update this description to better match what telegraf does for us?

File: README.md (hunk 2 above)
Comment (mboersma) [inline README.md:4]: Can we link to each of the three containers then?
[<sup>grafana</sup> Grafana Docker Repository on Quay](https://quay.io/repository/deis/grafana)&nbsp;&nbsp;[<sup>telegraf</sup> Telegraf Docker Repository on Quay](https://quay.io/repository/deis/telegraf)&nbsp;&nbsp;[<sup>influxdb</sup> InfluxDB Docker Repository on Quay](https://quay.io/repository/deis/influxdb)

Comment (jchauncey) [inline README.md:4]: they are linked in the sub directories. but i can add this to the main repo too

Comment (bacongobbler) [inline README.md:4]: that would be good. most people going to the repo aren't going to go into each subdirectory for image status. Having them all in the main README would be beneficial.

Comment (jchauncey) [inline README.md:4]: Well they will need to for config options and such. Didnt want to pollute the main readme with all that

Comment (mboersma) [inline README.md:4]: It's not strictly necessary or related to this PR, just a suggestion.

Comment (jchauncey) [inline README.md:4]: oh i did it. =)

**Pull request merged**